packages/tc-schema-sdk/data-accessors/graphql-defs.js

```diff
@@ -288,15 +296,19 @@ module.exports = {
 		scalar Date
 		scalar Time
 	`;
-		const typeDefinitions = types.map(printTypeDefinition);
+		const typeDefinitions = types
+			.filter(({ from, to }) => !from && !to)
+			.map(printTypeDefinition);
 
 		const relationshipTypeDefinitions = printRelationshipTypeDefinitions(
-			types,
+			this.getRelationshipTypes(retrieveOptions),
```

Comment:
does this still create relationship types even when their is not a user defined yaml file?

Reply:
Yes, `getRelationshipTypes` returns all resolved relationships in type properties with a bit more information e.g from, to, etc. Therefore we trust that `getRelationshipTypes()` returns _all_ relationships in user-defined type properties